backtrace-library/src/main/java/backtraceio/library/metrics/SummedEventsPayload.java

```diff
@@ -0,0 +1,20 @@
+package backtraceio.library.metrics;
+
+import com.google.gson.annotations.SerializedName;
+
+import java.util.concurrent.ConcurrentLinkedDeque;
+
+public class SummedEventsPayload extends EventsPayload<SummedEvent> {
+    @SerializedName("summed_events")
+    private final ConcurrentLinkedDeque<SummedEvent> summedEvents;
```

Comment:
` private final ConcurrentLinkedDeque<SummedEvent> summedEvents;` can be part of the EventsPayload abstract class - duplicated code.

Reply:
@konraddysput GSON does not have a nice way of handing this case, it doesn't support inheritance for `@SerializedName` annotation. The only alternative is creating a custom serializer, which defeats the whole point of using GSON for serializing for us. 
https://riptutorial.com/gson/example/22968/using-gson-with-inheritance
Getter won't help, GSON serializes based on the class structure, class needs to have members which can be serialized by GSON.